kolibri/plugins/learn/assets/src/views/MasteredSnackbars/index.vue

```diff
@@ -131,17 +131,17 @@
       iconBackgroundColor() {
         switch (this.nextContent.kind) {
           case ContentNodeKinds.EXERCISE:
```

Comment:
This is the third time I've seen this switch statement. Maybe this could be put into the theme mixin as a utility?

Reply:
agreed. Created a follow-up issue here: https://github.com/learningequality/kolibri/issues/5541